ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -58,6 +58,26 @@ ColumnLayout {
         readonly property var sourceModel: root.controller.sourceModel
         readonly property bool isLoading: root.isUpdating || root.isFetching
 
+        function toggleSortByDateIsDisabled(value) {
+            const orderByDateIndex =  cmbTokenOrder.indexOfValue(SortOrderComboBox.TokenOrderDateAdded)
+
+            cmbTokenOrder.model[orderByDateIndex].isDisabled = value
+            cmbTokenOrder.modelChanged()
+
+            if (!value && cmbTokenOrder.currentIndex === orderByDateIndex) {
+                cmbTokenOrder.indexOfValue(SortOrderComboBox.TokenOrderAlpha)
+            }
+        }
+
+        Component.onCompleted: {
+            settings.sync()
```

Comment:
Do we know where this file is located? What is its filename? Is it maintained per user?

Reply:
it has a standard location for each OS, for example on mac it should be in a place similar to ~/Library/Preferences/com.Status.plist or if specified other wise in the app.